### ISSUE TITLE
Multiline comments with double * throws Invalid Line Break

### ISSUE THREAD

**Issue by GrayedFox** [jscs-dev/node-jscs#1395]
I could not quite get a screenshot with the actual error message but managed to at least get it with the red error dialogue being shown.

To reproduce this simply enter the offending code snippet into the top of a .js file, save the file and then re-open your IDE. In my case: Phpstorm.

```
/**
 * Created by GrayedFox on 4/03/2015.
 * This script is called whenever the popup for the plugin is opened. It handles writing and restoring values to and from storage.
**/
```

Here is a screenshot for (semi) proof purposes:
![jscs_bug_validation](https://cloud.githubusercontent.com/assets/6493307/7731977/8a079272-ff24-11e4-88ce-d56dde42be42.PNG)

Update: don't be fooled by the screenshot - there is **no** space after that second astrix on the first line. Really.

**Comment by GrayedFox** [comment]
![jscs_bug_validation](https://cloud.githubusercontent.com/assets/6493307/7731969/8248083c-ff24-11e4-90a5-60e2ca1d6f81.PNG)

**Comment by markelog** [comment]
Could you run `jscs --verbose <path to problematic file>` with your jscs config?

**Comment by GrayedFox** [comment]
Text copy of the exact console output:

validateLineBreaks: Invalid line break at C:\Users\GrayedFox\WebstormProjects\QADR\updateVersions.js :
     1 |/**
-----------^
     2 | * Created by GrayedFox on 4/03/2015.
     3 | * This script is called whenever the popup for the plugin is opened. It handles writing and restoring values to and from storage.


1 code style error found.

**Comment by markelog** [comment]
Thank you

/cc @mikesherov

**Comment by markelog** [comment]
@GrayedFox hm, so there is no space, but there is a line break char, are sure you it is a correct one?

What is value of `validateLineBreaks` rule? Do you use some sort of plugin to create those comments?

**Comment by GrayedFox** [comment]
validateLineBreaks is set to LF (which is not Windows friendly?) which is, as far as I know, the only thing that gets unhappy here. I make those comments my self - just a style choice really for descriptive comments present at the beginning of a file.

Weird thing... If I indent the 3 lines that make up the comment, and then unindent them, or if I insert a new line character (press enter) at the end of the first line and then press backspace and delete the new line, the error goes away.

So I end up with the exact same comment lines and style, but with no error (until I reopen the file). Something to do with how JSCS reads the beginning of a file? 

![green_light](https://cloud.githubusercontent.com/assets/6493307/7836359/ddb774e0-047f-11e5-9fc9-08b121bc168b.PNG)

**Comment by GrayedFox** [comment]
PS Sorry for long reply time, was on holiday in Palermo.

**Comment by markelog** [comment]
> So I end up with the exact same comment lines and style, but with no error (until I reopen the file). Something to do with how JSCS reads the beginning of a file, which differs from how it reads a file which is edited/updated?

Nope, it seems this is an editor issue, i bet if you change `validateLineBreaks` value to `CRLF` this issue will go away, others should pop though :-).

You could try to open this file in vim, like so - http://stackoverflow.com/questions/3852868/how-to-make-vim-show-m-and-substitute-it, or show line breaks in some other way. If there really is `CRLF` char, then rule works correctly

**Comment by zxqfox** [comment]
Run `grep "\015" file` to see what lines with `CR` (`\r`)

And `tr -d "015" < file` to remove `\r`

**Comment by GrayedFox** [comment]
Ouch... 

Running ```grep "\015" ``` on it outputs 
> *  Created by GrayedFox on 8/05/2015.

So there is a /r character there. Why on earth it only appears there and not anywhere else is beyond me.

**Comment by GrayedFox** [comment]
Sorry guys - closing as the plugin is functioning as intended. Seems to be an issue with PhPStorm randomly injecting a /r character here.

**Comment by GrayedFox** [comment]
Oh god... this is embarrassing. I thought I had enforced LF line endings a long time ago... turns out my projects were still running CRLF instead (I think I changed default settings after staring both these projects?)

For anyone troubleshooting this type of error in the future and running a Jetbrains IDE - check the bottom right hand corner to see if you are using LF, CRLF or CR line endings (you probably want LF - check within your team). Blergh. *hides head in sand*
